app/Http/Controllers/User/SettingController.php

```diff
@@ -86,7 +97,10 @@ public function getPhoneSetting($phone, $radio)
                 'user_id' => $id,
             ];
             $displayPhone = $this->usermeta->getDataSetting($dataPhone);
-            $phones = $this->user->find($id)->get();
+            $data = [
+                'id' =>$id,
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Arrays.ArrayDeclaration.NoSpaceAfterDoubleArrow</br>**Severity**: error</br>**File**: app/Http/Controllers/User/SettingController.php</br>**Line**: 101</br>Expected 1 space between double arrow and "$id"; 0 found